tests/gpu/GPUHelpers.cpp

```diff
@@ -18,7 +19,7 @@
 namespace OCIO = OCIO_NAMESPACE;
 
 
-// TODO: Make OCIO::Platform::CreateTempFilename() public so it could be used here.
+// TODO: Make OCIO::Platform::CreateTempFile() public so it could be used here.

```

Comment:
Why CDLTransform_tests.cpp is able to use `OCIO::Platform::CreateTempFilename()` but not here ?

Reply:
The CPU unit tests are built using directly the cpp files from `OpenColorIO` to deeply test all classes/methods private to the the library (i.e. white box tests). That's critical as the CPU implementation is the reference to the other language translations. So, the GPU unit tests are built on top of the `OpenColorIO` public API (i.e. black box tests) and uses the CPU to validate the processing results. Note that a coming pull request implements the OSL translation validation using the same technic.

Currently the `OCIO::Platform::CreateTempFilename()` method is private. And the comment correctly explains the solution to be able using it here. I suggest to do it as the pull request revisits the implementation method.